### PR TITLE
feat: document first platform release 2023.11.10

### PR DESCRIPTION
**Pull request by womfoo**

### CODE REVIEW

**Comment by antonbaliasnikov** [comment]
@womfoo , could you, please, clarify what this is about? This repository is only for Atala platform releases. I believe the platform release can be in the repository that is related to the code when it's released. Or you would like to combine some infra changes here? We can also include them into our main ecosystem release.

**Comment by womfoo** [comment]
thanks for the review @antonbaliasnikov, the idea was to keep a logbook of the platform changes somewhere. Happy to make them inter the 2.6 release if it makes sense there (not sure since most of the platform changes are internal).

File: Atala Platform/2023.11.10.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Atala PRISM v2.5
```
Comment (mkbreuningIOHK) [inline Atala Platform/2023.11.10.md:1]: Change to Atala Platform, I guess no versioning as it is by date.

File: Atala Platform/2023.11.10.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+November 10 2023 release of the Atala Platform
+
+## Changelog
+
+**New Features:**
+
+- Environments Status Dashboard - A new dashboard showing all Atala 2.x Environments is now available at
+  https://atala.grafana.net/d/de0dedc4-0c84-4da8-980a-c69f521fd8ba/environment-status?orgId=1
+- ArgoCD Slack Notifications - Deployments status are now sent to internal slack channel #atala-release-notifications
```
Comment (mkbreuningIOHK) [inline Atala Platform/2023.11.10.md:11]: I would consider to add:
- a section that indicates what benefit the feature is adding
- a section that is asking the recipient of the release what is asked them to do with the new feature. Example on the environment dashboard, should the user click and connect to the dashboard and then what he/she can do with it? Same for the slack notification. Is there any feedback your are asking with this feature? (format of the info in the notification)
- a documentation section to this release template: if a user wants to know more about the added features, where to go

**Review by mkbreuningIOHK — CHANGES_REQUESTED** [review]

**Review by milosbackonja — APPROVED** [review]

**Comment by womfoo** [comment]
moving this as releases at https://github.com/input-output-hk/atala-prism-v2-infra

**Pull request closed without merging**